File: frontend/src/app/models/entity/Paramgrabartraspaso.ts
export class Paramgrabartraspaso{ //Se ingresan los campos que se usarán para la grabacion de la de-
    constructor (   //volución de la compra

      public idtraspaso        ?: number, //
      public servicioorigen    ?: number,
      public serviciodestino   ?: number, //
      public fpreexterno       ?: number,
      public fechatraspaso     ?: string,
      public respontraspaso    ?: string,
      public observtraspaso    ?: string,
      public tipomovimiento    ?: string,
      public bodegaorigen      ?: number,
      public bodegadestino     ?: number,
      public servidor          ?: string
      
    ) {}
}

File: backend/Modulos/models/paramGrabarTraspaso.go
package models

// ParamGrabarTraspaso is...
type ParamGrabarTraspaso struct {
	IDTraspaso      int    `json:"idtraspaso"`
	ServicioOrigen  int    `json:"servicioorigen"`
	ServicioDestino int    `json:"serviciodestino"`
	FpreExterno     int    `json:"fpreexterno"`
	FechaTraspaso   string `json:"fechatraspaso"`
	ResponTraspaso  string `json:"respontraspaso"`
	ObservTraspaso  string `json:"observtraspaso"`
	TipoMovimiento  string `json:"tipomovimiento"`
	BodegaOrigen    int    `json:"bodegaorigen"`
	BodegaDestino   int    `json:"bodegadestino"`
	PiUsuario       string `json:"usuario"`
	PiServidor      string `json:"servidor"`
}
